echam distrub recipe

pgierz opened the pull request:
How to apply an echam disturbance.

Probably, it would be nice to have the exp tree from a graphviz, but I didn't know how to do that.

pgierz committed:
echam distrub recipe

mandresm commented on docs/recipes/echam_disturbance.rst:
```diff
@@ -0,0 +1,73 @@
+   you can create a file called ``disturb_years.dat``. An abbreviated file tree
+   would look like::
+
+      * EXPID
+      |
+      | * scripts
+      | | - run_config.yaml
+      | | - distrub_years.dat
+      | |
+      | * outdata
+      | | - echam
+      | ... other files and folders ...
```
Do you mean to turn this into a tree in Graphviz? I can do that if you want.

pgierz replied:
That'd be nice!

mandresm commented on docs/cookbook.rst:
```diff
@@ -35,6 +35,7 @@ documentation issue on `our GitHub repository <https://github.com/esm-tools/esm_
 .. } (Move to the end of the paragraph)
 .. : sort (sort the lines you have selected alphabetically)
 
-.. include:: recipes/modify_namelists.rst
 .. include:: recipes/add_model_setup.rst
+.. include:: recipes/echam_disturbance.rst
+.. include:: recipes/modify_namelists.rst
 .. include:: recipes/use_own_namelist.rst
```
The names of the rst files do not correspond with the titles, and therefore, ordering them alphabetically by file name results in no order in the rendered version. I would suggest the following changes:
```
.. include:: recipes/echam_disturbance.rst       # Applying a Temporary...
.. include:: recipes/modify_namelists.rst           # Changing Namelist...
.. include:: recipes/add_model_setup.rst          # Implement a New...
.. include:: recipes/use_own_namelist.rst         # Using Your Own...
```

pgierz replied:
Reorganized based upon actual title

mandresm commented on docs/recipes/echam_disturbance.rst:
```diff
@@ -0,0 +1,73 @@
+Applying a temporary disturbance to ECHAM
```
Through out the document we follow the convention of capitalizing the everything that is not an article or a preposition. I am not the biggest fan of it but should we stick to it?

pgierz replied:
Christian now suggested an even longer title...Maybe we should find a nice system for this?

mandresm commented on docs/recipes/echam_disturbance.rst:
```diff
@@ -0,0 +1,73 @@
+namelist. As this is a common problem, there is a way to have the run do this
+automatically for some specific years of your simulation.
+
+1. Generate a file to list years you want distrubed.
```
there is a typo in `disturbed`

pgierz replied:
solved by christian

mandresm commented on docs/recipes/echam_disturbance.rst:
```diff
@@ -0,0 +1,73 @@
+
+
+2. Add years you want disturbed
+   The file should contain a list of years the distrubance should be applied
```
typo in `disturbance`

pgierz replied:
solved by christian

mandresm commented on docs/recipes/echam_disturbance.rst:
```diff
@@ -0,0 +1,73 @@
+  2007
+  2008
+
+You can also set the distrubance strength in your configuration under
```
typo in `disturbance`

pgierz replied:
solved by christian

mandresm commented on docs/recipes/echam_disturbance.rst:
```diff
@@ -0,0 +1,73 @@
+
+You can also set the distrubance strength in your configuration under
+``echam.disturbance``. The default is ``1.000001``. Here, we apply a 200%
+distrubance whenever a "distrub_year" is encountered.
```
typo in `disturbance` and `"disturb_year"`

pgierz replied:
solved by christian

christian-stepanek committed:
Update echam_disturbance.rst

Some updates for clarity. Thanks for the kind documentation of the feature, Paul.

pgierz committed:
Merge pull request #209 from christian-stepanek/patch-1

Update echam_disturbance.rst

pgierz committed:
Update cookbook.rst

alphabetical based on actual title

pgierz committed:
beautify ECHAM6 link
